recipe/recipe.yaml

```
@@ -32,15 +43,25 @@ requirements:
     - go-licenses
 
 tests:
+  - package_contents:
+      strict: true
+      files:
+        - bin/brev
+        - share/bash-completion/completions/brev
+        - share/zsh/site-functions/_brev
+        - share/fish/vendor_completions.d/brev.fish
+
   - script:
       - brev --version
+      - brev --help
+
 about:
   homepage: https://developer.nvidia.com/brev
   summary: 'Brev CLI'
   description: |
     CLI tool for managing workspaces provided by brev.dev
   license: MIT
-  license_file: 
+  license_file:
     - LICENSE
     - library_licenses/**/**/LICENSE
```

```suggestion
    - library_licenses/
```
I know this wasn't part of your original PR, but I believe we should specify the license directory like this because some of the license files can have a different name than `LICENSE`.